### ISSUE TITLE
Class generation from spec only when spec changed

### ISSUE THREAD

**Issue by c-schmitz-tsystems** [quarkiverse/quarkus-openapi-generator#878]
When I start my quarkus application with mvn quarkus:dev the openapi-generator from quarkus always generates the proxies from the swagger file, no matter if the spec changed or not. This triggers the compile and increases the startup time of the dev session.
It would be nice when the generator has a change detection which only generates the proxies in case of a changed swagger file.
Thanks for any comments on this

**Comment by ricardozanini** [comment]
Yes, that's a known issue since the inception of this extension. Feel free to send a PR to fix it; it's not on my list of priorities now. But I can help review a PR.

**Comment by c-schmitz-tsystems** [comment]
The openapi generator maven plugin supports such a feature but I cant reproduce the same client generation as the quarkiverse generator does. I there any plan to add a quarkus to it for rest client proxy generation?

**Comment by ricardozanini** [comment]
wdym? Sorry I'm not following.

> The openapi generator maven plugin supports such a feature but I cant reproduce the same client generation as the quarkiverse generator does. Is there any plan to add a quarkus generator to it for rest client proxy generation?

**Comment by michaelsonnleitner** [comment]
see also https://github.com/quarkiverse/quarkus-openapi-generator/issues/1536
i think for client it is closed now.

**Comment by ricardozanini** [comment]
I'm closing this ticket since we have implemented already see #1514
